ui/page/dcrdex/dcrdex_page.go

```diff
@@ -87,6 +97,12 @@ func (pg *DEXPage) HandleUserInteractions() {
 	if pg.CurrentPage() != nil {
 		pg.CurrentPage().HandleUserInteractions()
 	}
+	if pg.splashPageInfoButton.Button.Clicked() {
+		pg.showInfoModal()
+	}
+	if pg.navigateToSettingsBtn.Button.Clicked() {
+		pg.WL.AssetsManager.SetDexFirstVisit(true)
+	}
```

Comment:
I've seen this `navigateToSettingsBtn` a couple times and still don't know what it is. And looking at this click handler, it doesn't navigate to anywhere! Please rename the button to something that better describes what the button is meant to do.